libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -15,7 +15,9 @@
         class="sky-control-label"
         [attr.for]="dateRangePickerId + '-select-calculator'"
       >
-        {{ label || 'skyux_date_range_picker_default_label' | skyLibResources }}
+        {{
+          label || ('skyux_date_range_picker_default_label' | skyLibResources)
```

Comment:
This change was made because I realized we had an optimization problem here and were calling the resource pipe with the value of `label` at all times - which just returned the value of `label`. The extra parenthesis short circuits that so that the pipe is not called unnecessarily.